versioned_docs/version-v0.15/UserManuals/ConfigUI/Jenkins.md

```diff
@@ -51,14 +51,22 @@ Jenkins only supports `CI/CD` domain entities, transformed from Jenkins builds a
 - CI/CD: Jenkins builds, stages, etc.
 
 ### Step 3 - Adding Transformation Rules (Optional)
-
+Before you start, you should understand that 
 This set of configurations is used for calculating [DORA metrics](../DORA.md).
 
 If you're using Jenkins builds to conduct `deployments`, please select "Detect Deployment from Jenkins Builds", and input the RegEx in the following fields:
 
-- Deployment: A Jenkins build with a name that matches the given regEx will be considered as a deployment.
+- Deployment: A Jenkins build with its job name or a Jenkins build stage with its name that matches the given regEx will be considered as a deployment.
```

Comment:
Please replicate the changs to the v0.15 doc as well.

Reply:
fixed